src/app/shared/components/template/components/text.ts

```diff
@@ -54,6 +56,14 @@ export class TmplTextComponent extends TemplateBaseComponent implements OnInit {
   }
 
   getParams() {
+    if (
+      this._row.value === "undefined" ||
+      this._row.value === "NaN" ||
+      this._row.value === "null" ||
+      this._row.value === '""'
+    ) {
```

Comment:
It might be cleaner to specify all the options in a single array and then just check the array for the value, e.g.
```
if(["undefined","NaN","null",""].includes(this._row.value){
    this.isFalsy = true
}
```

Reply:
Or you could even make it a one-liner:
```
 this.isFalsy = ["undefined","NaN","null",""].includes(this._row.value)
```